radio/src/lua/api_general.cpp

```diff
@@ -482,13 +482,17 @@ The list of valid sources is available:
 * for OpenTX 2.1.x at http://downloads-21.open-tx.org/firmware/lua_fields.txt (depreciated)
 * for OpenTX 2.1.x Taranis and Taranis Plus at http://downloads-21.open-tx.org/firmware/lua_fields_taranis.txt
 * for OpenTX 2.1.x Taranis X9E at http://downloads-21.open-tx.org/firmware/lua_fields_taranis_x9e.txt
+* for OpenTX 2.2.x Taranis and Taranis Plus at http://downloads-22.open-tx.org/firmware/lua_fields_x9d.txt
+* for OpenTX 2.2.x Taranis X9E at  http://downloads-22.open-tx.org/firmware/lua_fields_x9e.txt
+* for OpenTX 2.2.x Horus at  http://downloads-22.open-tx.org/firmware/lua_fields_x12s.txt
 
 @param name (string) name of the field
 
 @retval table information about requested field, table elements:
  * `id`   (number) field identifier
  * `name` (string) field name
  * `desc` (string) field description
+ * 'unit' (number) unit identifier (https://opentx.gitbooks.io/opentx-2-2-lua-reference-guide/content/appendix/units.html)
```

Comment:
Proper link is `[units](../appendix/units.md)` and add text `added in 2.2.0`.

Also you should add to the `@status` section notice:
```
@status current Introduced in 2.0.8, filed `unit` added in 2.2.0
```